### ISSUE TITLE
Website doesnt display margin used labels

### ISSUE THREAD

**Issue by sliddjur** [evdubs/bfx-rates#1]
The Y axis doesnt have any labels for margin used. How much is "one bar"?

**Comment by evdubs** [comment]
Indeed. I have had it in mind to redo bfxrates, but it is very low on my list of priorities.
